### DISCUSSION TITLE
How to disable compression of a newly created ZIP?

### DISCUSSION THREAD

**Discussion by drauch** [gildas-lormeau/zip.js#428] [Q&A] [answered]
I just want to use the "archive" feature of ZIP and put some files into a ZIP, how to forego compression?

Best regards,
D.R.

**Comment by gildas-lormeau** [comment, marked as answer]
You have to set the option `level` to `0` when creating a `ZipWriter` instance or when calling the `add` method. 

See here for an example of code: https://github.com/gildas-lormeau/zip.js/blob/eb89a9c2448bd13637a86cb60c8784c33b5ae43f/tests/all/test-store.js#L15

See here for the documentation: https://gildas-lormeau.github.io/zip.js/api/interfaces/ZipWriterConstructorOptions.html#level.
